NetSuiteProject/Exercise.js

```diff
@@ -1,39 +1,34 @@
 //Add
-function add(x,y){
-	return x + y;
-};
+
 
 function addition(x,y){
-	if(typeof add (x,y) === "string"){
-	alert("This function only accepts Integers")
+	var add = (x + y);
+	if(typeof add === "string"){
+	alert("This function only accepts Numbers");
```

Comment:
Be meticulous and consistent with your spacing and indenting; the formatting of your code is essential to its readability by yourself and other developers. This `alert` is not indented within its surrounding `if`, which is misleading.